tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
From what I understand, tslint is deprectated and its possible to use eslint exclusively now
e.g. https://medium.com/@kaltepeter/migrating-from-tslint-to-eslint-f2cdf610c2e4

Reply:
This doesnt use tslint, but maybe you saw this very similar var name in the lint configs?

`import tseslint from 'typescript-eslint'`

the base config for eslint was made using `yarn create @eslint/config`, I just said 'yes' to typescript when prompted and thats what it added so I think its safe to say its recent and kosher.